source/guide/3.2/guide-security.md

```diff
@@ -364,7 +352,52 @@ In future versions, all communications from and to the Cloudify manager will uti
 
 # Writing your own userstore and authentication providers
 
-## how to write
+## Custom UserStore Implementation
+
+A valid userstore implementation can be any Python class that inherits from
+[abstract_userstore.py](https://github.com/cloudify-cosmo/flask-securest/blob/master/flask_securest/userstores/
+abstract_userstore.py) and implements: 
+
+- get_user() - returns a relevant user from the userstore. If a matching user is not found, returns None.<br>
+  The object returned by `get_user` must adhere to Flask-secuREST's [User Model](https://github.com/cloudify-cosmo/flask-securest/blob/master/flask_securest/models.py#L41).
+
+### LDAP UserStore Example:
+  An example for a userstore class - [LDAPUserStore](https://github.com/cloudify-cosmo/flask-securest/blob/master/flask_securest/userstores/examples/ldap_userstore.py).<br>
+  This class inherits from [AbstractUserstore](https://github.com/cloudify-cosmo/flask-securest/blob/master/flask_securest/userstores/abstract_userstore.py) and implements the `get_user` method as required. 
+
+  The properties to initialize this class should be specified in the manager blueprint as described earlier in [Configuring a Userstore](#configuring-a-userstore).<br>
+  For example:
+  {% highlight yaml %}
+  userstore_driver:
+    implementation: flask_securest.authentication_providers.ldap_userstore:LDAPUserStore
```

Comment:
shouldn't this be 
`flask_securest.userstores.examples.ldap_userstore:LDAPUserStore`?